diagnosis/getkuberneteslogs.sh

```diff
@@ -293,11 +304,24 @@ then
     
     CLUSTER_NODES=$(az vm list -g ${RESOURCE_GROUP} --show-details --query "[*].{Name:name,ip:privateIps}" --output tsv | grep 'k8s-' | cut -f 1)
     PROXY_CMD="ssh -i ${IDENTITYFILE} ${KNOWN_HOSTS_OPTIONS} ${USER}@${MASTER_IP} -W %h:%p"
-    
+
     for host in ${CLUSTER_NODES}
     do
         processHost ${host}
     done
+
+    #Get Windoews nodes log if Windows nodes exist
+    WINDOWS_NODES=$(az vm list -g ${RESOURCE_GROUP} --show-details --query "[*].{Name:name,ip:privateIps}" --output tsv | grep -v 'k8s-\|vmd-' | cut -f 1)
+
+    if [ -n "$WINDOWS_NODES" ]
+    then
+        echo "[INFO] Windows nodes detected, will collect log for windows nodes."
```

Comment:
I would remove this, not consistent with the linux nodes block

Reply:
Updated